File: WebArticles.WebAPI/Infrastructure/Models/PaginatorQuery.cs
using System.Collections.Generic;

namespace WebArticles.WebAPI.Infrastructure.Models
{
    public class PaginatorQuery
    {
        public RequestFilters Filters { get; set; }

        public string SearchString { get; set; } = "";
        public string SortBy { get; set; } = "id";
        public string SortDirection { get; set; } = "asc";
        public int Page { get; set; } = 1;
        public int PageSize { get; set; } = 10;
        
    }
}

File: WebArticles.Angular/src/app/data-model/infrastructure/models/paginator-query.ts
import { RequestFilters } from "./request-filters";

export class PaginatorQuery {
    filters: RequestFilters = new RequestFilters();
    searchString: string= "";
    page: number = 1;
    sortBy: string = "id";
    sortDirection: string = "asc";
    pageSize: number = 10;
}
